Cartfile.resolved

```diff
@@ -1,6 +1,7 @@
 github "AgileBits/onepassword-app-extension" "bcc4cc97fed9a6e73fa204f2e61138e353cb3ef7"
 github "DaveWoodCom/XCGLogger" "7.0.0"
 github "Leanplum/Leanplum-iOS-SDK" "2.4.2"
+github "Dev1an/A-Star" "3.0.0-beta-1"
```

Comment:
How come we need to pull in A-Star here? Or am I misunderstanding what "Cartfile.resolved" means

Reply:
Good question. AStar is dependancy of Mappamundi, and only the top-level dependencies are specified in the Cartfile. Once the Cartfile is processed, `Cartfile.resolved` is written out so that dependencies are pinned to specific tags or SHAs. The `Cartfile.resolved` will contain all the chained deps as well as the top-level ones. 
Another example of this is `application-services` has a child dep of `SwiftProtobuf`, so the latter will appear in the `Cartfile.resolved`.